sonar-text-dotnet/src/SonarLint.Secrets.DotNet.UnitTests/TestConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿/*
+ * Copyright (C) 2021-2023 SonarSource SA
+ * All rights reserved
+ * mailto:info AT sonarsource DOT com
+ */
+
+namespace SonarLint.Secrets.DotNet.UnitTests
+{
+    internal static class TestConstants
```

Comment:
Since the file is located in the Test project I find the prefix redundant. It can be `Constants`.